evm/src/main/java/com/horizen/evm/Evm.java

```diff
@@ -19,18 +19,19 @@ public static EvmResult Apply(
             BigInteger gasPrice,
             EvmContext context
     ) {
-        return LibEvm.evmApply(stateDBHandle.handle, from, to, value, input, gasLimit, gasPrice, context);
+        return LibEvm.evmApply(stateDBHandle.handle, from, to, value, input, gasLimit, gasPrice, context, false);
     }
 
-    public static EvmResult Apply(
+    public static EvmResult Trace(
```

Comment:
considering the comments below, we may have only `Apply` method with `boolean trace` arg there.